packages/graphpack/bin/graphpack.js

```diff
@@ -32,17 +33,33 @@ const createProductionBuild = ({ compiler }) => {
   });
 };
 
+const generateSchemaTypings = (watch = false) => {
+  const srcPath = process.env.GRAPHPACK_SRC_DIR || `${process.cwd()}/src/`;
+  const schemaFilesRegex = path.join(srcPath, './**/*.graphql');
+
+  generate({
+    schema: schemaFilesRegex,
+    template: 'graphql-codegen-typescript-template',
+    out: path.join(srcPath, 'graphql-typings.ts'),
+    args: [],
+    watch,
+    overwrite: true,
+  });
+};
+
 const startGraphPack = async () => {
   const config = await loadWebpackConfig();
   const compiler = webpack(config);
 
   require('yargs')
-    .command(['$0', 'dev'], 'Start graphpack dev server', {}, () =>
-      startDevServer({ compiler, config }),
-    )
-    .command('build', 'Create production build', {}, () =>
-      createProductionBuild({ compiler }),
-    ).argv;
+    .command(['$0', 'dev'], 'Start graphpack dev server', {}, () => {
+      generateSchemaTypings(true);
```

Comment:
I'm wondering if we should re-generate each time we have changes in the source?

Reply:
I think it's fine, because if the schema changes, the typings should update accordingly.